src/js/providers/html5.js

```diff
@@ -62,6 +62,9 @@ function VideoProvider(_playerId, _playerConfig) {
         },
         
         timeupdate() {
+            if (_positionBeforeSeek === _videotag.currentTime) {
+                return;
+            }
```

Comment:
Can this check be avoided? I noticed in the mixin we exit if `this.seeking` is true.

Reply:
@robwalch once seeking has ended, we're still in a buffering state but we get timeupdates where the current time has not changed. What that causes is the state being set to playing even though you're not done buffering, so the video stays frozen without  a buffer icon

Reply:
Seeking and buffering should be done on "seeked" where we set `this.seeking` to false.

Reply:
@robwalch buffering begins upon seeking, and continues after seeked
<img width="316" alt="screen shot 2017-10-31 at 12 35 30 pm" src="https://user-images.githubusercontent.com/12138319/32236679-3ed4eb20-be39-11e7-94c5-bf16758fc793.png">

waiting is fired right after seek, and the time update fired after seeked has not changed, therefore video has not progressed so video is still buffering.